README.md

```diff
@@ -149,6 +149,18 @@ Options include:
 	* Type: String
 	* Default: `#FFA66D`
 	* A hex color code to style Check It Out
+* refCount
+	* Type: Number
+	* Default: `500`
+	* Total number of references to show.
+
+To set `refCount` , start with the flag `--count`
+
+```
+checkit --count 20
+		OR
+checkit --count=20
```

Comment:
May not be necessary per my comments later in the review

Reply:
done